bokehjs/make/paths.ts

```diff
@@ -32,10 +32,6 @@ export const lib = {
     main: join(build_dir.lib, "main.js"),
     output: join(build_dir.js, "bokeh.js"),
   },
-  gl: {
-    main: join(build_dir.lib, "models/glyphs/webgl/main.js"),
-    output: join(build_dir.js, "bokeh-gl.js"),
-  },
```

Comment:
This is a breaking change, especially for anyone who uses bokehjs directly, so for backwards compatibility you have to keep an empty bundle until 3.0, similarly to how I kept `*.css` bundles pre 2.0.

Reply:
We could handle publishing an empty file in the release automation. I think that is the simplest option that only requires (future undo-able) changes in one place and keeps the main codebase cleaner. I will proceed that way later today unless you have other considerations to offer.

Reply:
Though I would also point out: If anyone is manually including BokehJS  bundles, then updating/changing versions at all will already require manual interventions, so I am not quite sure why another manual intervention to delete a bundle is any different from the user POV.

Reply:
> If anyone is manually including BokehJS bundles, then updating/changing versions at all will already require manual interventions, (...)

> We could handle publishing an empty file in the release automation.

There's an option of using bundles available via bokehjs' npm package, and those don't have version information. 

In general I'm not sure if care that much and the simpler the build the better. We can reconsider this before the release.

Reply:
> We can reconsider this before the release.

@mattpap Just to make sure I understand, you are agreeing that this can be merged now, as is?